beacon-chain/rpc/prysm/v1alpha1/beacon/blocks.go

```diff
@@ -39,6 +39,7 @@ func (bs *Server) ListBeaconBlocks(
 	if err != nil {
 		return nil, err
 	}
+	// TODO: figure out a better way to do this or change this API.
```

Comment:
Please resolve this TODO.

Reply:
it was added as a reminder i'll remove the comment and we can resolve it in a later PR